tools/clang/tools/dxcompiler/dxcompilerobj.cpp

```diff
@@ -1612,7 +1639,10 @@ class DxcCompiler : public IDxcCompiler3,
       Inlining = clang::CodeGenOptions::NormalInlining;
     compiler.getCodeGenOpts().setInlining(Inlining);
 
-    compiler.getCodeGenOpts().HLSLExtensionsCodegen = std::make_shared<HLSLExtensionsCodegenHelperImpl>(compiler, m_langExtensionsHelper, Opts.RootSignatureDefine);
+    compiler.getCodeGenOpts().HLSLExtensionsCodegen =
+        std::make_shared<HLSLExtensionsCodegenHelperImpl>(
+            compiler, m_langExtensionsHelper, Opts.RootSignatureDefine,
```

Comment:
An existing issue I noticed here is that we use `m_langExtensionHelper` here, but we are in `SetupCompilerForCompile` which takes a pointer to the helper as a parameter `helper`.  I think this code should use `*helper` instead.  I don't know if it makes a material difference with the current state of the code, but I think it's more _correct_ given the definition of this method.

Reply:
Probably no functional difference right now, but is cleaner. Fixed.